rulesets/data/denh_1_0_lookup.tsv

```diff
@@ -5,7 +5,7 @@ slab_perimeter_ins_depth		0	0	0	0	0	0	0	0	0	2	2	2	2	2	2	4	4	4	4	4
 slab_perimeter_ins_rvalue		0	0	0	0	0	0	0	0	0	10	10	10	10	10	10	10	10	10	10	10
 slab_under_ins_width		0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
 slab_under_ins_rvalue		0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
-floors_over_uncond_spc_ufactor		0.064	0.064	0.064	0.064	0.064	0.064	0.047	0.047	0.047	0.047	0.047	0.033	0.033	0.033	0.033	0.033	0.033	0.033	0.028	0.028
+floors_ufactor		0.064	0.064	0.064	0.064	0.064	0.064	0.047	0.047	0.047	0.047	0.047	0.033	0.033	0.033	0.033	0.033	0.033	0.033	0.028	0.028
```

Comment:
Don't we need floors_insulated rows in here too? It applies to both SF and MF units, right? https://github.com/NREL/OpenStudio-ERI/blob/denh/rulesets/resources/constants.rb#L52